diff --git a/node_modules/han-css/src/sass/inline/_jiya.sass b/node_modules/han-css/src/sass/inline/_jiya.sass
index 6d063b8..7916d38 100644
--- a/node_modules/han-css/src/sass/inline/_jiya.sass
+++ b/node_modules/han-css/src/sass/inline/_jiya.sass
@@ -70,7 +70,9 @@ h-char.bd-consecutive
     &:lang(zh-HK):before
       display: none
 
-  @at-root h-cs.jiya-outer[prev*='bd-cop'],
+  @at-root h-cs.jiya-outer[prev*='bd-cop']
+    display: none
+
   @at-root h-cs.jiya-outer.bd-end
     &:lang(zh-Hant),
     &:lang(zh-TW),
